### ISSUE TITLE
Fix internal bridge MTU

### ISSUE THREAD

**Issue by plajjan** [plajjan/vrnetlab#150]
TLDR; increase MTU on internal bridge to 10000

Long version:

Found a rather tricky issue today. I rebuilt my CI environment and all of sudden ISIS wasn't working. No adjacencies. WTF. Could swear this thing worked before and I haven't changed things in that sense... this CI test suite was running with SROS routers. Checked the one that is identical except for using XRv. Worked beautifully. Same with VMX.

Created a new test env with SROS and XRV devices. Wouldn't work. Tried turning hello-padding off.. which makes it work. Classic ISIS issue. So there's a MTU mismatch, how is that possible? This stuff worked before.

Spun up my old test environment that includes SROS, XRV and VMX. ISIS establishes just fine. Config pushed to these devices and the ones in the new environment is virtually identical except for identifiers like IPs etc.... hmm. What's the difference? The new environment uses multiple VMs in the  SROS machine - one VM per linecard and they are interconnected with a Linux bridge.... aaahhh! MTU 1500 on that bridge :/

We need to raise the MTU on such internal links to something high. The value doesn't even need to match because it's just L2 stuff from our perspective so the VMs will set their own MTU. Let's do like 10k.
